dotCMS/src/main/java/com/dotcms/contenttype/business/RelationshipFactory.java

```diff
@@ -23,6 +25,10 @@
 
     Relationship byTypeValue(final String typeValue);
 
+    Optional<Relationship> byParentChildRelationName(ContentType contentType, String relationName);
```

Comment:
Add Doc

Reply:
api wrapper method has doc so I think we are ok